community.hachyderm.io/content/en/blog/posts/2024-07-07-hachyderm-and-nivenly/index.md

```diff
@@ -0,0 +1,35 @@
+---
+date: 2024-07-07
+title: "Hachyderm and Nivenly"
+linkTitle: "Hachyderm and Nivenly"
+description: |
+  An update on the partnership between Hachyderm and Nivenly
+author: Hachyderm Moderation ([@Hachyderm](https://hachyderm.io/@Hachyderm))
+---
+Hey Hachyderm,
```

Comment:
Just a stylistic thing: most of our blog posts aren't written as letters (I wrote one as Exec Director last year to put out some fires, but that's the exception not the rule).